[06:20:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:20:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:03:55] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:03:55] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:47:17] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:47:17] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:30:53] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:30:53] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:14:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:14:20] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:57:55] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:57:55] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:41:36] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[10:41:36] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[11:25:41] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:25:41] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[12:09:46] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[12:09:46] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[12:43:45] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[12:43:45] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[12:53:51] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[12:53:51] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[13:26:58] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[13:26:58] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[13:37:07] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[13:37:07] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:10:25] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:10:25] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:20:52] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:20:52] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:54:02] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:54:02] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:04:19] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:04:19] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:37:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:37:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:47:30] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:47:30] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:21:06] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:21:06] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:31:10] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:31:10] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:04:34] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:04:34] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:14:35] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:14:35] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:48:40] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:48:40] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:58:35] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:58:35] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:32:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:32:39] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:42:31] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:42:31] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:16:25] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:16:25] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:26:14] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:26:14] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:00:14] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:00:14] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:10:08] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:10:08] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:44:03] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:44:03] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:53:54] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:53:54] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:27:54] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:27:54] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:37:45] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:37:45] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:11:41] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:11:41] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:21:32] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:21:32] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:55:31] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:55:31] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:05:25] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:05:25] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:24:39] Error: Telegram API Error  {
  code: 4,
  http_status_code: 500,
  message: 'Internal Server Error: restart'
}
[23:24:39] Error: Telegram API long poll problem. Code: 4
Internal Server Error: restart
[23:24:43] Error: Telegram API Error  { code: 4, http_status_code: 502, message: 'Bad Gateway' }
[23:24:43] Error: Telegram API long poll problem. Code: 4
Bad Gateway
[23:39:29] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:39:29] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:40:51] Error: Telegram API Error  {
  code: 4,
  http_status_code: 429,
  message: 'Too Many Requests: retry after 5'
}
[23:40:51] Error: Telegram API long poll problem. Code: 4
Too Many Requests: retry after 5
[23:49:23] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot4/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:49:23] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
